controls/def.cf

```diff
@@ -126,6 +126,14 @@ bundle common def
                           not(isvariable("trustkeysfrom"))),
         meta => { "defvar" };
 
+      "cf_serverd_maxconnections"
```

Comment:
OK, that doesn't work in 3.7.3 or 3.9.0

```
      ignore_missing_inputs => ifelse( isvariable("def.ignore_missing_inputs"),
                                                  $(def.ignore_missing_inputs),
                                       "false");

```

```
./promises.cf:12:47: error: Function does not return the required type
                                       "false");

```

Issue filed here: https://tracker.mender.io/browse/CFE-2400